geopandas/plotting.py

```diff
@@ -63,9 +63,16 @@ def _expand_kwargs(kwargs, multiindex):
     it (in place) to the correct length/formats with help of 'multiindex', unless
     the value appears to already be a valid (single) value for the key.
     """
+    import matplotlib
     from matplotlib.colors import is_color_like
     from typing import Iterable
 
+    if matplotlib.__version__ > LooseVersion("3.3.2"):
+        # alpha is supported as array argument with matplotlib 3.4+
```

Comment:
This is not the most robust version check (in case a 3.3.3 would get released), but not directly sure what to use to ensure we test it on CI with matplotlib master (has version '3.3.2.post1571+geeef6e593')

Reply:
If there'll be 3.3.3 it is an easy fix.

Reply:
Yes, but if this is already in a released version of geopandas, such a new version of matplotlib can break your code, which is annoying ..

Reply:
It looks that there will actually be 3.3.3 - https://github.com/matplotlib/matplotlib/pulls?q=is%3Aopen+is%3Apr+milestone%3Av3.3.3